install.sh

```diff
@@ -1,10 +1,11 @@
 #!/usr/bin/env bash
 set -o errexit
-set -o nounset
 set -o pipefail
 
 RELEASE_URL="https://api.github.com/repos/CircleCI-Public/circleci-cli/releases/latest"
-DEST="/usr/local/bin/circleci"
+DEST="${CIRCLECI_CLI_DEST:-/usr/local/bin/circleci}"
+
+set -o nounset
```

Comment:
Could be nicer to put all `set` commands back together here. But since `CIRCLECI_CLI_DEST` can purposedly be undefined this has to come after `DEST` defintion.

Reply:
Since you are setting a default value `nounset` will not fail on this line. It should be safe to move it back.